<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>239</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	font: 75 20pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>64</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(172, 172, 172);
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(172, 172, 172);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_degis">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>64</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(172, 172, 172);
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(172, 172, 172);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_yuzde">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>64</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(172, 172, 172);
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(172, 172, 172);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bolme">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>64</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(122, 122, 122);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(122, 122, 122);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>123</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>123</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>123</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_carp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>123</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(122, 122, 122);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(122, 122, 122);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_art">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(122, 122, 122);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(122, 122, 122);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eksi">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(122, 122, 122);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(122, 122, 122);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nokta">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(76, 76,76);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(76,76,76);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_esittir">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(122, 122, 122);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(122, 122, 122);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
